rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-download-from-google-drive.adoc

```diff
@@ -0,0 +1,168 @@
+= Download from Google Drive 
+
+Download or Export files from your Google Drive. 
+
+You can export Google Docs, Sheets, and Slides into a selection of supported formats including PDF.
+
+== Before You Begin 
+
+Before downloading a file, ensure you: 
+
+* Configured your Google API Service Account Credentials.
++
+You can configure your credentials as an xref:toolbox-variable-handling-activity-parameters.adoc#defining-activity-parameters[Activity Parameter] or inline in the action step wizard. 
+* Obtained the corresponding File ID by using the Search in Google Drive action step. 
+
+== Properties 
+
+* *Authentication Settings* 
+** *Auth Provider x509 Cert URL*
++
+Authentication provider certificate URL for the Google API credentials. 
+** *Auth URI* 
++
+OAuth2 authentication URI for the Google API credentials.
+** *Client Email* 
++
+Client email of the Google API credentials. 
+** *Client ID* 
++
+Client ID of the Google API credentials. 
+** *Client x509 Cert URL* 
++
+Client certificate URL for the Google API credentials. 
+** *Credential Type*
++
+Type of Google API credentials. For example, `service_account`.
+** *Google API Service Account Credentials* 
++
+Activity parameter containing all Google API credentials. 
+** *Project ID* 
++
+Project ID of the Google API credentials.
+** *Token URI* 
++
+OAuth2 token URI for the Google API credentials.
+** *Universe Domain* 
++
+Universe domain of the Google API credentials.
+
+* *Common* 
++
+See xref:common-properties.adoc[Common Properties]
++
+** *File ID*
++
+ID of the file to download or export.
+** *Google drive file type*
++
+Type of Google file to export. Available types are: 
++
+*** *Google Docs*
+*** *Google Sheets*
+*** *Google Slides*
++
+** *Operation mode*
++
+Operation to perform, either *DownloadFile* or *ExportDocument*.
+** *Docs type* 
++
+File format for exporting Google Docs. Available types: 
++
+*** *Microsoft Word (.docx)*
```

Comment:
Is the extension part of the UI element?

Reply:
Yes, all properties have the same name as they show in the UI.